sdk/eventhub/Azure.Messaging.EventHubs/tests/Amqp/AmqpMessageConverterTests.cs

```diff
@@ -12,6 +12,7 @@
 using Microsoft.Azure.Amqp.Framing;
 using NUnit.Framework;
 using NUnit.Framework.Constraints;
+using FramingData = Microsoft.Azure.Amqp.Framing.Data;
```

Comment:
Same deal.   I wouldn't consider it blocking, but I would definitely prefer that we don't need to do this.    At worst, can we use `Framing.Data` rather than aliasing?

Reply:
After looking into it and discussion, the clash was with `Azure.Data` namespace from `Azure.Data.SchemaRegistry`. We decided the best option was to keep the alias since partial qualification (`Framing.Data`) won't work here as we're not inside of a `Microsoft.Azure.Amqp` namespace.